dev/preview/workflow/preview/deploy-gitpod.sh

```diff
@@ -302,8 +302,8 @@ yq w -i "${INSTALLER_CONFIG_PATH}" experimental.ide.ideMetrics.enabledErrorRepor
 #
 # configureObservability
 #
-TRACING_ENDPOINT="http://otel-collector.monitoring-satellite.svc.cluster.local:14268/api/traces"
-yq w -i "${INSTALLER_CONFIG_PATH}" observability.tracing.endpoint "${TRACING_ENDPOINT}"
+#TRACING_ENDPOINT="http://otel-collector.monitoring-satellite.svc.cluster.local:14268/api/traces"
```

Comment:
Why is this disabled?

Reply:
Without monitoring enabled, server returns lots of errors in the logs.